x/plans/keeper/migrations.go

```diff
@@ -22,3 +25,39 @@ func (m Migrator) Migrate2to3(ctx sdk.Context) error {
 	}
 	return nil
 }
+
+func (m Migrator) Migrate3to4(ctx sdk.Context) error {
```

Comment:
2. We decided to put actual migration code in v3/... subdir if possible. In this case, following x/bank/... example, introduce `x/plans/migrations/v3/store.go:MigrateStore(...)` and call it from here. 
Because the migration code needs access to `keeper.plansFS`, maybe we can add an API in `x/plans/keeper/migrations.go` to expose it, with a clear comment stating the API should only be used in migrations>

Reply:
We decided to do that in the fixation migration code (which is a lib itself, not a module with keeper), and move the migration to separate subdirs if the effort is reasonable. Hence my suggestion to expose keeper.plansFS.